src/cli/ui/prompt.py

```
@@ -257,11 +265,13 @@ def _schedule_hide_message(self, app):
         """Schedule hiding the quit message after timeout."""
 
         def hide():
-            self._show_quit_message = False
-            self._last_ctrl_c_time = None
+            self._reset_ctrl_c_state()
             app.invalidate()
 
-        app.loop.call_later(self._ctrl_c_timeout, hide)
+        try:
+            app.loop.call_later(self._ctrl_c_timeout, hide)
+        except Exception:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)